Workade/Views&ViewModels/Magazine/TipDetail/TipDetailViewController.swift

```diff
@@ -8,25 +8,79 @@
 import UIKit
 
 class TipDetailViewController: UIViewController {
-    let sampleLabel: UILabel = {
-        let label = UILabel()
-        label.text = "Tip 뷰 입니다."
-        label.textColor = .black
-        label.translatesAutoresizingMaskIntoConstraints = false
+    // TODO: 임시 데이터 -> 추후 요청으로 수정
+    let titleArray = ["내 성격에 맞는\n장소 찾는 법", "바다마을에서\n보낸 일주일", "워케이션\n경험자의 조언", "워케이션\n경험자의 꿀팁", "워케이션\n경험자의 특별한장소", "워케이션\n센터주변 맛집정보"]
+    
+    private let tipDetailCollectionView: UICollectionView = {
+        let flowLayout = UICollectionViewFlowLayout()
+        flowLayout.scrollDirection = .vertical
+        
```

Comment:
```suggestion
        flowLayout.minimumLineSpacing = 20
```
[P1] 셀 간 여백 설정을 20으로 맞춰주시면 감사하겠습니다!!

Reply:
반영 하였습니다 :)